### ISSUE TITLE
Code Exporting

### ISSUE THREAD

**Issue by NomNuggetNom** [chiquitinxx/grooscript-grails3-plugin#12]
I'm not sure this is totally possible, so bear with me. Let's say we have a simple utility class on the back-end:

```
package util

class Util {
   static String shout(s) { 
      s.toUppercase()
   }
}
```

And let's say I want to auto-magically use this script on the front end, as such:
```
<grooscript:code>
   def s = util.Util.shout('Hello, world!')
</grooscipt:code>
```

In order to get this variable bound in the grooscript context, a simple annotation is introduced, `@ExportScript`. Simply tagging the class with this annotation will result it in being present on the front-end.

**Comment by chiquitinxx** [comment]
So, imagine you annotate classes with @ExportScript, then what plugin do? Add that js converted code to all gsp's with grooscript tags?

Can you please explain a bit more this improvement, it seems nice idea!

**Comment by NomNuggetNom** [comment]
Yes @chiquitinxx exactly :). The classes annotated would be usable within the `grooscript:code` tag
